docs/docs/behaviors/backlight.md

```diff
@@ -9,6 +9,10 @@ This page contains [backlight](../features/backlight.mdx) behaviors supported by
 
 ## Backlight Action Defines
 
+:::info
```

Comment:
Instead of these extra info boxes, I'd just remove the references to necessary headers.

Reply:
Apologies for the delay. Removed the info boxes and reduced the section to simply listing the defines as a table for all of the documentation pages touched in this PR.